src/lib/abacus/index.ts

```diff
@@ -450,8 +450,12 @@ class AbacusStateManager {
    * https://linear.app/dydx/project/feature-experimentation-6853beb333d7/overview
    */
   setStatsigConfigs = (statsigConfig: { [key in StatSigFlags]?: boolean }) => {
-    const { [StatSigFlags.ffSkipMigration]: useSkip = false } = statsigConfig;
+    const { [StatSigFlags.ffSkipMigration]: useSkip = false, ...rest } = statsigConfig;
     StatsigConfig.useSkip = useSkip;
+    Object.entries(rest).forEach(([k, v]) => {
+      // @ts-ignore
+      StatsigConfig[k] = v;
```

Comment:
@jaredvu we can do this as long as we are careful to keep the names consistent between abacus and v4-web (which again, we can probably ensure by creating an enum or something in abacus and exporting it to web)